web/files/referencedby_test.go

```diff
@@ -34,6 +35,12 @@ func TestAddReferencedByOneRelation(t *testing.T) {
 		return
 	}
 
+	var result struct {
+		Data struct {
+			Rev          string                 `json:"rev"`
+			ReferencedBy []couchdb.DocReference `json:"referencedBy"`
+		}
```

Comment:
I'm really surprised that it works with `` `json:"data"` ``. I didn't think the JSON unmarshalling was case-insensitive.

Reply:
Ah. Well, that's unexpected :). I actually copied this struct from another test.
I can add the json tag explicitly if you think this could become a problem.